### PR TITLE
`declaration-block-no-ignored-properties` now ignore `display: inline-block` with `float` property.

### PR DESCRIPTION
**Pull request by evilebottnawi**
Close https://github.com/stylelint/stylelint/issues/1238
/cc @davidtheclark @jeddy3

### CODE REVIEW

**Comment by davidtheclark** [comment]
The same must be done for `inline`, right?

**Comment by evilebottnawi** [comment]
@davidtheclark hm, need check in css spec

**Comment by jeddy3** [comment]
> Close #1238

This is the last remaining `7.0.0` issue (woohoo!). I got the impression from https://github.com/stylelint/stylelint/issues/1238#issuecomment-219236893 that finer-grained control might be needed. I just wanted to confirm that this has been accounted for in this (or the previous PRs for this rule), and that this final PR should close this issue.

**Comment by davidtheclark** [comment]
Please correct me if I'm wrong, but my understanding is that the approach of this PR is to ignore the pairing, rather than correcting it.

**Comment by evilebottnawi** [comment]
@davidtheclark yes now we ignore all values where use `float: *`

File: src/rules/declaration-block-no-ignored-properties/README.md [hunk 1]
```diff
@@ -13,7 +13,6 @@ Certain property value pairs rule out other property value pairs, causing them t
 The rule warns when it finds:
 
 - `display: inline` used with `width`, `height`, `margin`, `margin-top`, `margin-bottom`, `float`, `overflow` (and all variants).
```
Comment (davidtheclark) [inline src/rules/declaration-block-no-ignored-properties/README.md]: This line needs to be changed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,7 @@
 - Changed: `time-no-imperceptible` now checks vendor prefixed properties.
 - Changed: `selector-attribute-quotes` now checks attribute selectors with whitespace around the operator or inside the brackets.
 - Changed: plugin arrays in extended configs are now concatenated with the main config's plugin array, instead of being overwritten by it. So plugins from the main config and from extended configs will all be loaded.
+- Changed: `declaration-block-no-ignored-properties` now ignore `display: inline-block` and `display: inline` with `float` property.
```
Comment (davidtheclark) [inline CHANGELOG.md]: Let's say: "`declaration-block-no-ignored-properties` now ignores the combination of `float` and `display: inline-block | inline`."

**Comment by davidtheclark** [comment]
LGTM.

**Comment by evilebottnawi** [comment]
/cc @jeddy3 
I think we can start release 7.0 :tada:

**Pull request merged**